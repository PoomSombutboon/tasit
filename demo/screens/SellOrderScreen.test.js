import React from "react";
import NavigationTestUtils from "react-navigation/NavigationTestUtils";
import { shallow } from "enzyme";
import SellOrderScreen from "./SellOrderScreen";

describe("SellOrderScreen", () => {
  jest.useFakeTimers();
  beforeEach(() => {
    NavigationTestUtils.resetInternalState();
  });

  it("renders the component", async () => {
    const sellOrder = {
      id: -1,
      priceMana: 0,
      priceUsd: 0,
      asset: {
        id: -1,
        name: "Not found",
        img: "https://decentraland.org/images/logo-65f7b27caf.png",
      },
    };
    const navigation = { getParam: () => sellOrder };
    expect(
      shallow(<SellOrderScreen navigation={navigation} />)
    ).toMatchSnapshot();
  });
});
