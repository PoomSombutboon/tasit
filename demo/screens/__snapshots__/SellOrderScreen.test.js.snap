// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SellOrderScreen renders the component 1`] = `
<SellOrderClaim
  onClaim={[Function]}
  sellOrder={
    Object {
      "asset": Object {
        "id": -1,
        "img": "https://decentraland.org/images/logo-65f7b27caf.png",
        "name": "Not found",
      },
      "id": -1,
      "priceMana": 0,
      "priceUsd": 0,
    }
  }
/>
`;
